AudioKit/Common/Internals/Microtonality/AKTuningTable+EqualTemperament.swift

```diff
@@ -7,25 +7,31 @@
 //
 
 extension AKTuningTable {
-
-    // Default tuning table is 12ET.
-    public func defaultTuning() {
-        twelveToneEqualTemperament()
-    }
-
-    public func twelveToneEqualTemperament() {
-        equalTemperament(notesPerOctave: 12)
-    }
-
-    public func thirtyOneEqualTemperament() {
-        equalTemperament(notesPerOctave: 31)
-    }
-
-    public func equalTemperament(notesPerOctave npo: Int) {
-        var nf = [Frequency](repeatElement(1.0, count: npo))
-        for i in 0 ..< npo {
-            nf[i] = Frequency(pow(2.0, Frequency(Frequency(i) / npo)))
-        }
-        tuningTable(fromFrequencies: nf)
+  
```

Comment:
I think there might be different white-space formatting between the iOS and MacOS Xcode projects